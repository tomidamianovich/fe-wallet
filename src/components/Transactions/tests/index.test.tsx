import { render } from '@testing-library/react';
import Transaction from '../index';
import { createStore, applyMiddleware } from "redux";
import { Provider } from 'react-redux';

const store = createStore(()=>[], applyMiddleware());

test('renders Transaction with balances', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Transaction />
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot();
});