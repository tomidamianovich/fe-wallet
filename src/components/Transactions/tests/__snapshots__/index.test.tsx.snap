// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Transaction with balances 1`] = `
<DocumentFragment>
  <div
    class="fe-wallet__transactions"
  >
    <span>
      Mis movimientos
    </span>
    <div
      class="fe-wallet__transactions__container"
    >
      <table>
        <thead>
          <tr>
            <th>
              Fecha
            </th>
            <th>
              Tipo de Operación
            </th>
            <th>
              Método
            </th>
            <th>
              Estado
            </th>
            <th>
              Comisión
            </th>
            <th>
              Monto
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
  </div>
</DocumentFragment>
`;
